modules/testing/pages/integration/snapshots/index.adoc

```diff
@@ -0,0 +1,94 @@
+= Snapshots
+
+// Optional: Define context for cross-references if needed within the module structure
+// :context: snapshots
+
+A {ProductName} **Snapshot** represents a specific, immutable collection of container images for the Components within an Application at a particular point in time. They capture the exact versions of all relevant component artifacts that were built and intended to be used together including their Git commits.
```

Comment:
Should we move the information from the [What is a Snapshot section](https://github.com/konflux-ci/docs/blob/4508c8bcd473d004426fa97b0f3c4fbe1d6468bd/modules/testing/pages/integration/snapshots/working-with-snapshots.adoc?plain=1#L5-L31) here?

Reply:
Yes, good call! I just pushed this with a new commit.

Reply:
@dirgim , how does it look now?

Reply:
Looks good, thanks